contracts/CO30_Matias_LivingInTheShadows.Tanoa/edit_me/briefing.sqf

```diff
@@ -0,0 +1,78 @@
+#include "..\do_not_edit\script_component.hpp"
+/*
+ * Author: Kresky, Jonpas
+ * Adds briefing based on side.
+ * Call from initPlayerLocal.sqf.
+ * Copy to other side briefings if you want to add briefings to those sides.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call FUNC(briefing);
+ *
+ * New Line:
+ * <br/>
+ *
+ * Text Colour (Hex colour Code)
+ * <font color='#FF0000'>TEXT HERE</font>
+ *
+ * Clickable Marker Link:
+ * <marker name = 'MARKER_NAME'>MARKER_TEXT</marker>
+ *
+ * Image:
+ * <img image='FOLDER\IMAGE.jpg' width='200' height='100'/>
+ */
+
+params ["_player"];
+
+switch (side _player) do {
+    // BLUFOR briefing
+    case west: {
+        
+    };
+
+    // OPFOR briefing
+    case east: {
+
+    };
+
+    // RESISTANCE/INDEPENDENT briefing -- THIS IS THE BRIEFING YOU WANT TO EDIT FOR YOUR MISSION!
+    case resistance: {
+        _player createDiaryRecord ["Diary", ["Employer", "
+            <br/>Pan-Chem, a subsidiary of the Pan-Global corporation
+        "]];
+
+        _player createDiaryRecord ["Diary", ["Situation", "
+            <br/><font color='#D81717'>ENEMY FORCES:</font>
+            <br/>Unknown
+            <br/>Unknown
+            <br/>
+            <br/><font color='#1D69F6'>FRIENDLY FORCES:</font>
+            <br/>Synixe Contractors
+            <br/>
+            <br/><font color='#993399'>CIVILIAN CONSIDERATIONS:</font>
+            <br/>N/A
+            <br/>
+            <br/><font color='#663300'>TERRAIN CONSIDERATIONS:</font>
+            <br/>The AO is covered by a thick jungle
+        "]];
+
+        _player createDiaryRecord ["Diary", ["Mission", "
+            <br/>Recently Synixe intercepted a distress signal from a facility deep in the Tanoan jungle belonging to the Pan-Chem corporation.
```

Comment:
Either that is the official title "The Pan-Chem Corporation", or it should just be "to Pan-Chem" imo, being nit-picky but it does stand out to me

Reply:
It's just to point out the name of the corporation